src/realm/sync/changeset.cpp

```diff
@@ -58,6 +58,7 @@ InternString Changeset::find_string(StringData string) const noexcept
     return InternString{};
 }
 
+#pragma runtime_checks( "", off )
```

Comment:
@simonask this "fixes" the stack corruption issue I'm seeing in the .NET tests, but seems a bit like a nuclear option - let me know if you would rather see this addressed in another way.